src/main/java/edu/illinois/cs/dt/tools/minimizer/MinimizeTestsResult.java

```diff
@@ -151,7 +151,7 @@ public Result expected() {
     }
 
     public List<String> withDeps() {
-        final List<String> order = new ArrayList<>(deps());
+        final List<String> order = new ArrayList<>(polluters.get(0).deps());    // TODO: What should this return?
```

Comment:
We should make sure to handle the case where there are no polluters here.

Otherwise, I think this is a reasonable solution for what this method does, although it may make more sense to add another method as well to perform the function this method was used for (rewriting). We could have it return a list of lists of all orders with all the polluters that we know of, then we could try the diffing/rewriting strategy on each of these lists separately. Not sure that's the best approach, feel free to let me know what you think.

Reply:
Actually, I'm not sure I understand completely what is intended for this method. Right now, I'm just trying to match the API in returning some set of deps, namely the first one.